charts/internal/alicloud-infra/values.yaml

```
@@ -1,26 +1,27 @@
 alicloud:
   region: cn-beijing
 
-create:
-  vpc: true
-
 clusterName: test-namespace
 
 sshPublicKey: sshkey-12345
 
 vpc:
+  create: true
   id: alicloud_vpc.vpc.id
   cidr: 10.10.10.10/6
+
+natGateway:
   natGatewayID: alicloud_nat_gateway.nat_gateway.id
```

now that its parent is already `natGateway`

```suggestion
  id: alicloud_nat_gateway.nat_gateway.id
```